lib/iris/analysis/_scipy_interpolate.py

```diff
@@ -43,8 +43,6 @@ def _ndim_coords_from_arrays(points, ndim=None):
     return points
 
 
-# source: https://github.com/scipy/scipy/blob/b94a5d5ccc08dddbc88453477ff2625\
-# 9aeaafb32/scipy/interpolate/interpolate.py#L1400
```

Comment:
Why have you removed this comment?

Reply:
Because it seemed appropriate.  It's starting to get really difficult to work out what I'm looking at or doing in IntelliJ, so my strategy is: make some changes, push them up so I can see them properly, then figure out if that was a good idea or not.

I can see now that removing that line was not a good idea because it will still be relevant, so I'll pop that back in.